<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>View</class>
 <widget class="QMainWindow" name="View">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>744</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>view</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_31">
    <item>
     <widget class="QGraphicsView" name="graphicsView"/>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_24">
        <item>
         <widget class="QPushButton" name="videoStartButton">
          <property name="text">
           <string>Start</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_26">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_27">
          <item>
           <widget class="QLabel" name="videoInputLabel">
            <property name="text">
             <string>From:</string>
            </property>
            <property name="buddy">
             <cstring>fromLineEdit</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="videoInputLineEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="videoFileOpenButton">
          <property name="text">
           <string>Browse...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_28">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_29">
          <item>
           <widget class="QLabel" name="videoOutputLabel">
            <property name="text">
             <string>To:</string>
            </property>
            <property name="buddy">
             <cstring>toLineEdit</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="videoOutputLineEdit"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTextEdit" name="videoFileListTextEdit"/>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>298</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_30"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>744</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_Project"/>
    <addaction name="actionImport_Image_File"/>
    <addaction name="separator"/>
    <addaction name="actionSave_Image"/>
    <addaction name="actionExit_program"/>
   </widget>
   <widget class="QMenu" name="menuBlack_White_process">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionGreyscale_single_image"/>
    <addaction name="actionPele_Mele"/>
    <addaction name="actionGrayscale_all_images"/>
    <addaction name="separator"/>
    <addaction name="actionFirst_Plan"/>
    <addaction name="actionBackground"/>
    <addaction name="actionErase_Image"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuBlack_White_process"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
   <addaction name="actionNew_Project"/>
   <addaction name="actionImport_Image_File"/>
   <addaction name="separator"/>
   <addaction name="actionGrayscale_all_images"/>
   <addaction name="actionPele_Mele"/>
   <addaction name="separator"/>
   <addaction name="actionSave_Image"/>
   <addaction name="actionErase_Image"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionImport_Image_File">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Icons/Icons/1481087480_gnome-app-install.png</normaloff>:/Icons/Icons/1481087480_gnome-app-install.png</iconset>
   </property>
   <property name="text">
    <string>Import Image File</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionExit_program">
   <property name="text">
    <string>Exit program</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionGreyscale_single_image">
   <property name="text">
    <string>Greyscale single image</string>
   </property>
   <property name="toolTip">
    <string>actionGreyscale_single_image</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="actionSelect">
   <property name="text">
    <string>Select</string>
   </property>
   <property name="shortcut">
    <string>S</string>
   </property>
  </action>
  <action name="actionFirst_Plan">
   <property name="text">
    <string>First Plan</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionErase_Image">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Icons/Icons/Actions-trash-empty-icon.png</normaloff>:/Icons/Icons/Actions-trash-empty-icon.png</iconset>
   </property>
   <property name="text">
    <string>Erase Image</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionSave_Image">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Icons/Icons/1481087980_document-save.png</normaloff>:/Icons/Icons/1481087980_document-save.png</iconset>
   </property>
   <property name="text">
    <string>Save Image</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionPele_Mele">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Icons/Icons/1481087628_image_add.png</normaloff>:/Icons/Icons/1481087628_image_add.png</iconset>
   </property>
   <property name="text">
    <string>Pele Mele</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionNew_Project">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Icons/Icons/Action-view-remove-icon.png</normaloff>:/Icons/Icons/Action-view-remove-icon.png</iconset>
   </property>
   <property name="text">
    <string>New Project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionGrayscale_all_images">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Icons/Icons/1481087601_editor-image-picture-photo-glyph.png</normaloff>:/Icons/Icons/1481087601_editor-image-picture-photo-glyph.png</iconset>
   </property>
   <property name="text">
    <string>Grayscale all images</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+G</string>
   </property>
  </action>
  <action name="actionSave_all_Images">
   <property name="text">
    <string>Save all  Images</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionBackground">
   <property name="text">
    <string>Background</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
